Source/Models/SwishError.swift

```diff
@@ -2,47 +2,69 @@ import Foundation
 import Argo
 
 public enum SwishError {
-  case ArgoError(Argo.DecodeError)
-  case DeserializationError(NSError)
-  case ParseError(NSError)
-  case ServerError(code: Int, data: NSData?)
-  case URLSessionError(NSError)
+  case argoError(Argo.DecodeError)
+  case deserializationError(Error)
+  case parseError(Error)
+  case serverError(code: Int, data: Data?)
+  case urlSessionError(Error)
 }
 
-extension SwishError: ErrorType { }
+extension SwishError: Error { }
 
-public extension SwishError {
-  var rawError: NSError {
+extension SwishError: CustomNSError {
+  public static let errorDomain = "com.thoughtbot.swish"
+
+  public var errorCode: Int {
+    switch self {
+    case .argoError:              return 1
+    case .deserializationError:   return 2
+    case .parseError:             return 3
+    case .serverError:            return 4
+    case .urlSessionError:        return 5
```

Comment:
Removing the aligment felt pretty messy and hard to scan for me. I ended up putting the returns statements on separate lines instead.